### ISSUE TITLE
empty pdf file if only one paragraph in addpdfresources branch in lua mode.

### ISSUE THREAD

**Issue by AlexandrKozlovskiy** [u-fischer/tagpdf#28]
When in tex document we have only one paragraph,instead of alt/actualtext nvda read,that there iis a blank. When in document more then one paragraph or if we use generic mode,all works as expected.
[issue_of_one_paragraph.txt](https://github.com/u-fischer/tagpdf/files/3801005/issue_of_one_paragraph.txt)

**Comment by u-fischer** [comment]
Yes, I saw this also: nvda sometimes ignores the /Alt text. This seems to be an nvda bug and should be reported there.

**Comment by AlexandrKozlovskiy** [comment]
But this happends only in lua mode,so probably problems with pdf.

**Comment by u-fischer** [comment]
The pdfs are different, e.g. in generic mode your example is not fully tagged as the footer is not marked as artifact. nvda seems no to like if a formula is the only thing on a page.

**Comment by AlexandrKozlovskiy** [comment]
But I have the same problem,if in mc i have usual text,instead of formula,and it's happends only if i 
[issue_of_one_paragraph.zip](https://github.com/u-fischer/tagpdf/files/3801889/issue_of_one_paragraph.zip)
have in lua mode only one paragraph.

**Comment by AlexandrKozlovskiy** [comment]
After the latest commit problem in lua mode was not fixed.

**Comment by u-fischer** [comment]
Show your tex and the resulting pdf. Be aware that this branch is work-in-progress and depends on work-in-progress in the pdfresource branch - it is quite possible that it creates currently broken pdf in some cases.

**Comment by AlexandrKozlovskiy** [comment]
[issue_of_paragraph.zip](https://github.com/u-fischer/tagpdf/files/3804096/issue_of_paragraph.zip)

**Comment by u-fischer** [comment]
I think I didn't push yesterday night correctly. I have done it now. I also adapted the package date, so that you can identify the version. It should say

~~~~
Lua module: tagpdf 2019-11-04 0.70 tagpdf lua code)
~~~~

**Comment by AlexandrKozlovskiy** [comment]
Thank you very much,now it seems works ok,at least for usual text.

**Comment by AlexandrKozlovskiy** [comment]
Should i recreate associated_files pr,to syncronize your changes in addpdfresources branch?

**Comment by u-fischer** [comment]
No that's not needed, part of it will have to go to other places, so I will only use the ideas not the actual code.
